app/main.py

```diff
@@ -2,0 +1,31 @@
+import os
+
+
+def move_file(command: str) -> None:
+    components = command.split()
+    first_part, second_part, third_part = components
+    if len(components) == 3 and first_part == "mv":
+
+        file_name = second_part
+        path = third_part
+        dirs = third_part.split("/")
+
+        if len(dirs) > 1:
+            current_path = dirs[0]
+            os.makedirs(current_path, exist_ok=True)
```

Comment:
Using `os.makedirs(current_path, exist_ok=True)` is appropriate here to handle existing directories without raising an error, which aligns with the checklist requirement to avoid errors when directories already exist.